source/dubregistry/registry.d

```diff
@@ -290,6 +294,25 @@ class DubRegistry {
 		m_db.setPackageRepository(pack_name, repository);
 	}
 
+	void setPackageLogo(string pack_name, NativePath path)
+	{
+		auto png = generateLogo(path);
+		if (png.length)
+			m_db.setPackageLogo(pack_name, png);
+		else
+			throw new Exception("Failed to generate logo");
+	}
+
+	void unsetPackageLogo(string pack_name)
```

Comment:
FWIW this is typically called delete (though I don't want to get into this nitpicking)